docs/netserver/config/csssprite.md

```diff
@@ -1,23 +1,29 @@
 ---
 title: CssSprite element
 uid: ns_config_csssprite
-description: NetServer CssSprite element
-so.date: 06.06.2018
-author: {github-id}
-keywords: config
+description: NetServer configuration for the Web client CssSprite generator.
+so.date: 12.07.2021
+author: Bergfrid Dias
+keywords: config, NetServer, web.config, CssSprite, Enabled, ExcludePaths, MaxHeight, MaxWidth, PersistentPath, RootPath
```

Comment:
This is my last comment about keywords... All files with more than 5 need to be changed.